server/src/test/java/org/elasticsearch/plugins/PluginsUtilsTests.java

```diff
@@ -368,6 +368,53 @@ public void testJarHellSpiConflict() throws Exception {
         assertThat(e.getCause().getMessage(), containsString("DummyClass1"));
     }
 
+    public void testStableEarlierElasticsearchVersion() throws Exception {
+        PluginDescriptor info = new PluginDescriptor(
+            "my_plugin",
+            "desc",
+            "1.0",
+            Version.fromId(Version.CURRENT.id + 1),
```

Comment:
Since all we care about changing is the version, a utility for creating a descriptor with a specific Version would reduce a lot of boilerplate in these tests. Before the only case to change the version was in `testIncompatibleElasticsearchVersion()`, but this adds 2 more, so I would combine them to make the tests easier to read. Something like `newTestDescriptorForVersion(Version v)`